frontend/src/main/components/BasicCourseSearch/BasicCourseSearchForm.js

```diff
@@ -18,20 +18,49 @@ const BasicCourseSearchForm = ({ setCourseJSON, fetchJSON }) => {
   const localQuarter = localStorage.getItem("BasicSearch.Quarter");
   const localLevel = localStorage.getItem("BasicSearch.CourseLevel");
 
-  const firstDepartment = allTheSubjects[0].subjectCode;
+  const loadObjectToAxiosParams = () => ({
+    url: "/api/UCSBSubjects/load",
+    method: "POST",
+    params: {},
+  });
```

Comment:
Also I think we don't need to load in the course search form, only need to get. Loads called from another page. See pages/adminLoadSubjectsPage.js

Reply:
Yup removed it

Reply:
I lied we need the load api to load in the subjects to the repository